app/src/main/java/com/example/chrono/main/stopwatch/StopwatchFrag.kt

```diff
@@ -29,6 +29,7 @@ class StopwatchFrag : Fragment() {
     private var lapCount = 0
     private var prevTime = 0.toLong()
     private lateinit var laps: ArrayList<LapObject>
+    private var maxLapReached = 0
```

Comment:
make this a bool u fool

Reply:
you can also do private lateinit var maxLapReached and initialize the var inside the onCreate() function. doesn't make sense to initialize it here and do the same thing in onCreate()